include/crow/http_connection.h

```diff
@@ -404,6 +406,8 @@ namespace crow
             {
                 do_write_general();
             }
+
+            queue_length -= 1;
```

Comment:
Could be replaced with `queue_length --`

Reply:
Done! I'm used to -= / += in my projects...